src/ripple/app/ledger/LedgerTiming.h

```diff
@@ -74,6 +74,11 @@ const int LEDGER_IDLE_INTERVAL = 15;
 // it takes to adjust the close time accuracy window
 const int LEDGER_VAL_INTERVAL = 300;
 
+// The number of seconds a validation remains current after the time we first
+// saw it. This provides faster recovery in very rare cases where the number
+// of validations produced by the network is lower than normal
+const int LEDGER_VAL_LOCAL = 180;
+
```

Comment:
`LEDGER_VAL_LOCAL` is too generic a name. `LEDGER_VAL_INTERVAL` deals with sign time, and `LEDGER_VAL_LOCAL` deal with seen time. How about renaming these `LEDGER_INTERVAL_{SIGN,SEEN}`?